dist/jstat.js

```diff
@@ -921,12 +921,13 @@ jStat.pooledstdev = function pooledstdev(arr) {
 
 // mean deviation (mean absolute deviation) of an array
 jStat.meandev = function meandev(arr) {
-  var devSum = 0;
-  var mean = jStat.mean(arr);
-  var i;
-  for (var i = arr.length - 1; i >= 0; i--)
-    devSum += Math.abs(arr[i] - mean);
-  return devSum / arr.length;
+  var med = jStat.median(arr);
+  var a = [];
+  for (var i = arr.length - 1; i >= 0; i--) {
+    a.push(Math.abs(arr[i] - med));
+  }
+  a.sort(function(a, b) { return a - b });
```

Comment:
I think this is unnecessary, since [median itself sorts the array](https://github.com/jstat/jstat/blob/master/src/vector.js#L119).